lib/engine/game/g_18_norway/game.rb

```diff
@@ -539,6 +539,19 @@ def emergency_issuable_bundles(corp)
 
           []
         end
+
+        def sell_movement(corporation = nil)
+          return self.class::SELL_MOVEMENT if corporation.nil?
```

Comment:
can avoid `nil?` call:

```suggestion
          return self.class::SELL_MOVEMENT unless corporation
```

Reply:
Looks better